spec/interactors/notes/create_spec.rb

```diff
@@ -0,0 +1,16 @@
+require "rails_helper"
```

Comment:
You can test interactor organizer this way.
```ruby
describe Notes::Create do
  describe ".organized" do
    let(:expected_interactors) do
      [
        Notes::UpdateSlug
      ]
    end

    subject { described_class.organized }

    it { is_expected.to eq(expected_interactors) }
  end
end
```

If the logic is complex and we need to test that the interactors work together correctly, we can add this code after `describe ".organized" do` block:
```ruby
describe ".call" do
  subject(:create_note) { Notes::Create.call(note: note) }
  
  let(:note) { build :note, user: user, title: "Lorem ipsum", content: "Dolor sit amet" }
  let(:user) { create :user }

  let(:expected_attributes) do
    {
      note: title: "Lorem ipsum",
      slug: "lorem_ipsum",
      content: "Dolor sit amet",
      user_id: user.id
    }
  end

  it "saves user" do
    expect { create_note }.to change(Note, :count).by(1)
    expect(note).to be_persisted
    expect(note).to have_attributes(expected_attributes)
  end
end
```